chain/client/src/client.rs

```diff
@@ -1640,27 +1639,6 @@ impl Client {
             if let Err(err) = self.send_network_chain_info() {
                 error!(target: "client", ?err, "Failed to update network chain info");
             }
-
-            // If the next block is the first of the next epoch and the shard
-            // layout is changing we need to reshard the transaction pool.
-            // TODO make sure transactions don't get added for the old shard
-            // layout after the pool resharding
```

Comment:
ditto - it's likely still needed for IR. Maybe even without changes but needs checking. I'll add an item to look into that to the tracking issue.